.travis.yml

```diff
@@ -14,3 +14,7 @@ install:
 script:
   # - cd tests/unit/ # unnecessary
   - pytest
+
+# bypass submodules because gitlab requires authentication for pull
+git:
+  submodules: false
```

Comment:
I don't understand how that took you 2h yesterday lol when we knew that was the fix

Reply:
I ran into other issues :eyes: